Makefile

```diff
@@ -70,9 +70,10 @@ test-skopeo:
 	@export GOPATH=$$(mktemp -d) && \
 		skopeo_path=$${GOPATH}/src/github.com/containers/skopeo && \
 		vendor_path=$${skopeo_path}/vendor/github.com/containers/image && \
-		git clone -b $(SKOPEO_BRANCH) https://github.com/$(SKOPEO_REPO) $${skopeo_path} && \
-		rm -rf $${vendor_path} && cp -r . $${vendor_path} && rm -rf $${vendor_path}/vendor && \
-		cd $${skopeo_path} && \
+		git clone https://github.com/$(SKOPEO_REPO) $${skopeo_path} && \
+		cd $${skopeo_path} && git checkout $(SKOPEO_BRANCH) && \
+		sed -i 's/FROM fedora/FROM fedora:30/g' Dockerfile && \
```

Comment:
The tests are failing due to
```
gpg: agent_genkey failed: No pinentry
gpg: key generation failed: No pinentry
```
And `fedora:30` is newer than this code. At least this particular failure can be avoided by going back to `:29`.